<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tbMain">
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideMiddle</enum>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>890</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="action_Save_as"/>
    <addaction name="separator"/>
    <addaction name="action_Export_CSV"/>
    <addaction name="separator"/>
    <addaction name="action_Settings"/>
   </widget>
   <widget class="QMenu" name="menu_Connection">
    <property name="title">
     <string>&amp;Connection</string>
    </property>
    <widget class="QMenu" name="menuMeasure">
     <property name="title">
      <string>Measure</string>
     </property>
     <addaction name="action_Start_measure"/>
     <addaction name="action_Pause_measure"/>
     <addaction name="action_Cancel_measure"/>
    </widget>
    <addaction name="action_Connect"/>
    <addaction name="separator"/>
    <addaction name="menuMeasure"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="action_Zoom_in"/>
    <addaction name="action_Zoom_out"/>
    <addaction name="action_Zoom_to_screen"/>
    <addaction name="separator"/>
    <addaction name="action_Points_labels"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menu_Connection"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="action_Open"/>
   <addaction name="action_Save"/>
   <addaction name="action_Save_as"/>
   <addaction name="separator"/>
   <addaction name="action_Points_labels"/>
   <addaction name="separator"/>
   <addaction name="action_Connect"/>
   <addaction name="separator"/>
   <addaction name="action_Start_measure"/>
   <addaction name="action_Pause_measure"/>
   <addaction name="action_Cancel_measure"/>
   <addaction name="separator"/>
   <addaction name="action_Zoom_in"/>
   <addaction name="action_Zoom_out"/>
   <addaction name="action_Zoom_to_screen"/>
   <addaction name="separator"/>
   <addaction name="action_Settings"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Open">
   <property name="icon">
    <iconset>
     <normaloff>:/more24/Open_24x24.png</normaloff>:/more24/Open_24x24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset>
     <normaloff>:/more24/Save_24x24.png</normaloff>:/more24/Save_24x24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Save_as">
   <property name="icon">
    <iconset>
     <normaloff>:/more24/Copy_24x24.png</normaloff>:/more24/Copy_24x24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_New">
   <property name="icon">
    <iconset>
     <normaloff>:/more24/New_24x24.png</normaloff>:/more24/New_24x24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Settings">
   <property name="icon">
    <iconset>
     <normaloff>:/more24/Settings_24x24.png</normaloff>:/more24/Settings_24x24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Settings</string>
   </property>
   <property name="toolTip">
    <string>Settings</string>
   </property>
  </action>
  <action name="action_Connect">
   <property name="icon">
    <iconset>
     <normaloff>:/24/lightning.png</normaloff>:/24/lightning.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect</string>
   </property>
  </action>
  <action name="action_Start_measure">
   <property name="icon">
    <iconset>
     <normaloff>:/more24/Play_24x24.png</normaloff>:/more24/Play_24x24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Start</string>
   </property>
   <property name="toolTip">
    <string>Start measure</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Return</string>
   </property>
  </action>
  <action name="action_Pause_measure">
   <property name="icon">
    <iconset>
     <normaloff>:/more24/Pause_24x24.png</normaloff>:/more24/Pause_24x24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Pause</string>
   </property>
   <property name="toolTip">
    <string>Pause measure</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Space</string>
   </property>
  </action>
  <action name="action_Cancel_measure">
   <property name="icon">
    <iconset>
     <normaloff>:/more24/Stop_24x24.png</normaloff>:/more24/Stop_24x24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Cancel</string>
   </property>
   <property name="toolTip">
    <string>Cancel measure</string>
   </property>
  </action>
  <action name="action_Zoom_out">
   <property name="icon">
    <iconset>
     <normaloff>:/more24/Zoom Out_24x24.png</normaloff>:/more24/Zoom Out_24x24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Zoom out</string>
   </property>
   <property name="toolTip">
    <string>Zoom out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="action_Zoom_in">
   <property name="icon">
    <iconset>
     <normaloff>:/more24/Zoom In_24x24.png</normaloff>:/more24/Zoom In_24x24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Zoom in</string>
   </property>
   <property name="toolTip">
    <string>Zoom in</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="action_Zoom_to_screen">
   <property name="icon">
    <iconset>
     <normaloff>:/more24/Zoom To_Screen_24x24.png</normaloff>:/more24/Zoom To_Screen_24x24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Zoom to plot</string>
   </property>
   <property name="toolTip">
    <string>Zoom to plot</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="action_Export_CSV">
   <property name="icon">
    <iconset>
     <normaloff>:/24/file-excel.png</normaloff>:/24/file-excel.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Export CSV</string>
   </property>
   <property name="toolTip">
    <string>Export data to .csv</string>
   </property>
  </action>
  <action name="action_Points_labels">
   <property name="icon">
    <iconset>
     <normaloff>:/24/tag.png</normaloff>:/24/tag.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Points labels</string>
   </property>
   <property name="toolTip">
    <string>Toogle points labels</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset>
     <normaloff>:/24/sign-info.png</normaloff>:/24/sign-info.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="toolTip">
    <string>About application</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
